scss/main.scss

```diff
@@ -282,6 +289,91 @@ body {
   height: 5%;
 }
 
+// *** what can you do section ***
+a:hover {
+  text-decoration: none;
+}
+
+.what-can-you-do {
+  .nav {
+    display: flex;
+    justify-content: center;
+  }
+  .nav-tabs {
+    border-bottom: none;
+    margin-bottom: 0.5rem;
+    justify-content: space-evenly;
+  }
+  .tab-content {
+    width: 100%;
+    height: 20rem;
+    background-color: $ex-light-lavender;
+    border-radius: 33px;
+    padding: 2rem;
+  }
+  .title {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+  .tab {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
since the tab labels aren't the same length, the section looks a bit off-center - could you adjust to make "learn new skills" sit in the middle